fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -2883,6 +2891,19 @@ public Builder setMetaDataStore(@Nullable FDBMetaDataStore metaDataStore) {
             return this;
         }
 
+        @Override
+        @Nullable
+        public Descriptors.FileDescriptor getLocalMetaDataFileDescriptor() {
+            return localMetaDataFileDescriptor;
+        }
+
+        @Override
+        @Nonnull
+        public Builder setLocalMetaDataFileDescriptor(@Nullable Descriptors.FileDescriptor localMetaDataFileDescriptor) {
+            this.localMetaDataFileDescriptor = localMetaDataFileDescriptor;
+            return this;
```

Comment:
The `localMetaDataFileDescriptor` still isn't being used to construct the `RecordMetaData` when using an `FDBMetaDataStore`.